src/App.tsx

```diff
@@ -10,6 +10,7 @@ import CreateLayout from 'Routes/Onboarding/Layout'
 import CreateAccount from 'Routes/Onboarding/CreateAccount'
 import ImportAccount from 'Routes/Onboarding/ImportAccount'
 import AddressBook from 'Routes/AddressBook'
+import AddressBookDetails from 'Routes/AddressBookDetails'
```

Comment:
AddressBookDetails is hard to understand component name. May be better ContactDetails or AddressDetails?